### PR TITLE
feat: add DirectoryOrCreate in k8s-deploy.yaml

### PR DESCRIPTION
**Pull request by ppborah**
Required directory creation in k8s-workers through k8s-deploy.yaml

### CODE REVIEW

File: k8s-deploy.yaml [hunk 1]
```diff
@@ -204,6 +204,10 @@ spec:
             - name: config
               mountPath: "/app/.env"
       volumes:
+        - name: config
```
Comment (klesh) [inline k8s-deploy.yaml:207]: The name is conflicting with the following one,

**Review by klesh — COMMENTED** [review]
Good job.
An minor issue is to be addressed

**Comment by ppborah** [comment]
PS: Making some changes, please don't merge!

**Review by klesh — APPROVED** [review]
LGTM

**Comment by ppborah** [comment]
Thanks for the PR merge! 🎉

**Comment by klesh** [comment]
Thanks for your contribution 😃

**Pull request merged**